e2e/cypress/integration/ad_ldap/saml_ldap_sync_id_attrib_spec.js

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @ldap @saml
+
+import {getAdminAccount} from '../../support/env';
+import {getRandomId} from '../../utils';
+
+// assumes that E20 license is uploaded
+// assumes openldap docker available on config default http://localhost:389
+// assumes keycloak docker - uses api to update
+// assumes the CYPRESS_* variables are set (CYPRESS_keycloakBaseUrl / CYPRESS_keycloakAppName)
+// requires {"chromeWebSecurity": false}
+// copy ./mattermost-server/build/docker/keycloak/keycloak.crt -> ./mattermost-server/config/saml-idp.crt
+describe('ad_ldap', () => {
+    const admin = getAdminAccount();
+    const baseUrl = Cypress.config('baseUrl');
+    const loginButtonText = 'SAML';
+
+    const user3 = {
+        username: 'e2etest.three.saml',
+        password: 'Password1',
+        email: 'e2etest.three@mmtest.com',
+        firstname: 'FirstSaml',
+        lastname: 'ThreeSaml',
+        ldapfirstname: 'TestLDAP',
+        ldaplastname: 'ThreeLDAP',
+        keycloakId: '',
+    };
+    const users = {user3};
+
+    const {
+        keycloakBaseUrl,
+        keycloakAppName,
+    } = Cypress.env();
+
+    const idpUrl = `${keycloakBaseUrl}/auth/realms/${keycloakAppName}/protocol/saml`;
+    const idpDescriptorUrl = `${keycloakBaseUrl}/auth/realms/${keycloakAppName}`;
+
+    const newConfig = {
+        SamlSettings: {
+            Enable: true,
+            EnableSyncWithLdap: false,
+            EnableSyncWithLdapIncludeAuth: false,
+            Verify: true,
+            Encrypt: false,
+            SignRequest: false,
+            IdpUrl: idpUrl,
+            IdpDescriptorUrl: idpDescriptorUrl,
+            IdpMetadataUrl: '',
+            ServiceProviderIdentifier: `${Cypress.config('baseUrl')}/login/sso/saml`,
+            AssertionConsumerServiceURL: `${Cypress.config('baseUrl')}/login/sso/saml`,
```

Comment:
nit, can use `baseUrl` directly. Same with other instance.
```suggestion
            ServiceProviderIdentifier: `${baseUrl}/login/sso/saml`,
            AssertionConsumerServiceURL: `${baseUrl}/login/sso/saml`,
```